src/config/dark-sites.config

```diff
@@ -479,6 +480,7 @@ m4rtyr.github.io
 macbb.org
 magnetoo.io
 magoware.tv
+mainstre.am
```

Comment:
Doesn't seem that dark mode to me.

![image](https://user-images.githubusercontent.com/25481501/129266328-7679251d-
a412-4608-8da8-965f6d7acf11.png)